app/kumactl/cmd/install/render_helm_files.go

```diff
@@ -38,7 +38,10 @@ metadata:
 `, namespace)
 }
 
-func renderHelmFiles(templates []data.File, args interface{}, kubeClientConfig *rest.Config) ([]data.File, error) {
+// When Kuma chart is embedded into other chart all the values need to have a prefix. You can set this prefix with this var.
+var HELMValuesPrefix = ""
```

Comment:
Maybe we should start isolating all these global variables in dedicated `interface.go` files so we can track them?